ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitter.scala

```diff
@@ -169,8 +170,25 @@ private[kvutils] class TransactionCommitter(
                     RejectionReason.InvalidLedgerTime(reason))),
               _ => StepContinue(transactionEntry)
             )
-        }
-        .getOrElse(StepContinue(transactionEntry))
+        case _ => // Pre-execution: propagate the time bounds and defer the checks to post-execution
```

Comment:
```suggestion
        case None => // Pre-execution: propagate the time bounds and defer the checks to post-execution
```
I try to avoid wildcard case branches such that the compiler can warn me if the patterns are not exhaustive.

Reply:
Yes, I think it's a safe policy; I was a bit torn between being more concise and being explicit in this case. In the end I went for conciseness because I don't expect we'll be branching the patterns further but it's very cheap extra safety, so I'll do as you suggest.

Reply:
Done